crates/turborepo-lib/src/run/task_id.rs

```diff
@@ -9,7 +9,7 @@ pub const ROOT_PKG_NAME: &str = "//";
 
 /// A task identifier as it will appear in the task graph
 #[derive(Debug, Clone, Hash, PartialEq, Eq, PartialOrd, Ord, Serialize)]
-#[serde(from = "String")]
+#[serde(from = "String", into = "String")]
```

Comment:
I have not seen this directive and it's un-googleable. No need for a comment in the code but what does this do?

Reply:
When serializing this type, `serde` will first convert it into the type specified by `into`: https://serde.rs/container-attrs.html#into